Doc/c-api/module.rst

```diff
@@ -451,8 +451,8 @@ state:
 
    On success, return ``0``. On error, raise an exception and return ``-1``.
 
-   Return ``NULL`` if *value* is ``NULL``. It must be called with an exception
-   raised in this case.
+   Return ``-1`` if *value* is ``NULL``. In this case, the function raises an
+   exception if called without an exception raised.
```

Comment:
I find the second sentence a bit convoluted, but I think it is nice to mention that an exception can be raised. Perhaps @AlexWaygood has a good suggestion for a rewording? :)

Reply:
(I'm on vacation this week, but I'll try to take a look when I get back :)

Reply:
Resolved as outdated; will start a new thread on the latest changes.